packages/lib/src/components/CustomCard/types.ts

```diff
@@ -1,28 +1,174 @@
-import { CardConfiguration } from '../Card/types';
-import { SFError } from '../Card/components/CardInput/types';
+import { CardBrandsConfiguration } from '../Card/types';
+import { Placeholders, SFError } from '../Card/components/CardInput/types';
 import UIElement from '../internal/UIElement';
+import {
+    CbObjOnAllValid,
+    CbObjOnBinLookup,
+    CbObjOnBinValue,
+    CbObjOnBrand,
+    CbObjOnConfigSuccess,
+    CbObjOnFieldValid,
+    CbObjOnFocus,
+    CbObjOnLoad,
+    StylesObject
```

Comment:
I think we could also fix part of the #2968 issue by exposing these Custom Card types.

For that, we would need to add to the [main exposed type file](https://github.com/Adyen/adyen-web/blob/main/packages/lib/src/types.ts) something like:

```js
export { CbObjOnAllValid, etc..  } from './components/internal/SecuredFields/lib/types';
```



I'd propose also to rename these types to match how we have typed for the main callbacks at the [core/types.ts](https://github.com/Adyen/adyen-web/blob/main/packages/lib/src/core/types.ts#L222) (onSubmit, onAdditionalDetails, etc.. ) . Meaning that `CbObjOnBinLookup` would be renamed to `BinLookupData` , `CbObjOnAllValid` to `AllValidData` and so on..  That matches with our current exposed types

Reply:
> I think we could also fix part of the https://github.com/Adyen/adyen-web/issues/2968 issue by exposing these Custom Card types.

??
This is exactly the issue that this PR is fixing.
If you run `yarn build` and look at `es/index.d.ts` file you will see that ,with this change, all the types (`CbObjOnAllValid`, `CbObjOnBinLookup` etc)  are now visible

Reply:
It is there but it is not being exported. You need to do expose it as I mentioned above

Reply:
ok, got it!